### PR TITLE
Move `test/contrib` to `test/scapy/contrib`

### PR DESCRIPTION
**Pull request by polybassa**
This PR attempts to move test/contrib into test/scapy/contrib

### CODE REVIEW

File: test/configs/windows2.utsc [hunk 1]
```diff
@@ -2,12 +2,12 @@
   "testfiles": [
```
Comment (gpotter2) [inline test/configs/windows2.utsc:2]: Could you delete this file. It wrongly resurected

Comment (polybassa) [reply test/configs/windows2.utsc]: done

**Comment by gpotter2** [comment]
Not sure why this staled TBH but guedou has been handling all tests migration.

**Comment by polybassa** [comment]
Well.. no problem. Since I'm recently made some PRs which moved around stuff between files, I figured out, that it is probably the best approach, to make a new PR and immediately merge it. Otherwise intermediate merges will maybe not be respected and changes get lost. Which means, such a move need to be done by a maintainer, and immediately merged, in my opinion.

**Pull request closed without merging**